docs/understanding-tracking-design/versioning-your-data-structures/cloud/index.md

```diff
@@ -5,14 +5,14 @@ sidebar_label: "Using the Data Structures Builder"
 sidebar_position: 20
 sidebar_custom_props:
   offerings:
-    - cloud
```

Comment:
This should still be `enterprise` and `cloud`. It’s not available in `community`.